conans/client/command.py

```diff
@@ -961,8 +968,8 @@ def run(self, *args):
             logger.error(exc)
             errors = True
         except ConanException as exc:
-            # import traceback
-            # logger.debug(traceback.format_exc())
+            import traceback
+            logger.debug(traceback.format_exc())
```

Comment:
I know. Just forgot to comment lines again. I usually use them during development.